Dronuts/common/models/drone.js

```diff
@@ -1,16 +1,4 @@
 'use strict';
-// var app = require('../../server/server');
 
 module.exports = function(Drone) {
-    Drone.validatesPresenceOf('drone_id');
```

Comment:
I understand that persistent models add ids themselves, but does this prohibit us from doing things like "1a", "1b" , etc like we initially wanted by making them strings?

Reply:
Yes.